ocaml/lambda/lambda.mli

```diff
@@ -714,8 +715,13 @@ and lfunction = private
     loc : scoped_location;
     mode : alloc_mode;     (* alloc mode of the closure itself *)
     ret_mode: alloc_mode;
-    region : bool;         (* false if this function may locally
-                              allocate in the caller's region *)
+    (** The mode of the return value. If [local], the function might allocate in
+        caller's region. *)
+    region : bool;
+    (** Does the function have a region? Even if this is [true], the function
+        can still allocate in caller's region via [Lexclave].
+
+        CR zqian: this field is always true and should be removed. *)
```

Comment:
I think your CR comment is not actually accurate: there are some code paths that set it to `false` and I believe they are still reachable. It should still be removed though.

Reply:
My understanding of `region` is wrong. I will move those comments to #2985 instead.